shared/depositutil/deposit.go

```diff
@@ -1,12 +1,14 @@
-// Package keystore contains useful functions for dealing
```

Comment:
Created this new package `depositutil` which is no longer tied to the geth keystore shared package. It also has been updated to use our latest bls code

Reply:
Great!